src/realm/sync/noinst/client_reset.cpp

```diff
@@ -329,8 +329,30 @@ void transfer_group(const Transaction& group_src, Transaction& group_dst, util::
         }
     }
 
-    converters::EmbeddedObjectConverter embedded_tracker;
+    // We must re-create any missing objects that are abscent in dst before trying to copy
```

Comment:
*absent

Reply:
missing objects or objects that are absent

Reply:
Whoops.